axelor-production/src/main/java/com/axelor/apps/production/service/app/AppProductionServiceImpl.java

```diff
@@ -63,4 +67,14 @@ public int getNbDecimalDigitForBomQty() {
 
     return DEFAULT_NB_DECIMAL_DIGITS;
   }
+
+  @Override
+  @Transactional
+  public void updatePartnerSubcontractTag() {
+    List<Partner> partnerList = partnerRepo.all().filter("self.isSubcontractor IS TRUE").fetch();
+    for (Partner partner : partnerList) {
+      partner.setIsSubcontractor(false);
+      partnerRepo.save(partner);
+    }
```

Comment:
This could cause issue with a large number of partners, it is better to instead execute an UPDATE query directly using JPA.em().createQuery()